### ISSUE TITLE
On a VPS ??!!!

### ISSUE THREAD

**Issue by GrinCa** [davidmann4/ogar-feeder-bot#381]
Hello I'm french but I gonna try to speak english !!

Is it possible to run the cheat on a VPS because being on my computer as I do not have a good connection I have not many bots (8/9 max). if anyone knows how then do I want to know, thank you !!

**Comment by hassanila97** [comment]
Yes its possible... it has been asked 1 or 2 times already.   Check other issues and youll get the andwer

**Comment by hassanila** [comment]
Yes its possible... it has been asked 1 or 2 times already.   Check other issues and youll get the answer

**Comment by GrinCa** [comment]
Indeed there is already post on that but no precise answer. I know manage with a VPS but the problem is the script on chrome.

**Comment by GrinCa** [comment]
Besides I know this is possible because AgarBots.net sells these bots so you have to put that cheat on a server.

**Comment by hassanila** [comment]
Whats the problem ?    Change the ip on the userscript and also on the config.    On config to 127.0.0.1   and userscript to your server ip.   You also need to do port forwarding.... you need to be able to access that port ( default 8081)  Search on Google/Youtube how to port forward and do that on your server.

**Comment by GrinCa** [comment]
sorry to be boring but could you detail me the operations to follow. Just put any file on the server and what data change? thank you ;)

**Comment by GrinCa** [comment]
Can you help me please ?

**Comment by philw1508** [comment]
@hassanila97 i have a question too... I port forwarded the remotedesktop connection to port 3389.... is it better to do port 8081?

**Comment by GrinCa** [comment]
@philw1508  Have you managed to put the cheat on a VPS. If yes, how ? thank you ;)

**Comment by hassanila** [comment]
@philw1508 No, 3389 is for RDP and its the best for it so let it be so..  for feeder you need to port forward 8081 also.     @Pako98 sry but I helped a lot of ppl with that and I dont have anymore time for it... sry  Som1 else can hlp you

**Comment by GrinCa** [comment]
@hassanila97 Thanks anyway

**Comment by GrinCa** [comment]
@philw1508 Could you help me if you know how to do please ?

**Comment by VoOoLoX** [comment]
- Clone the repo to the VPS
- Change IP in the config file to the VPS's IP (for windows vps open cmd and type ipconfig, for linux vps open terminal and type ifconfig)
- Tweak the settings (change your UUID,add proxies, etc)
- On the client side change the IP in user script to match VPS's public IP

--I'd suggest you to make a script to restart feeder server (because bots slowly disconnect after some time)
--Most VPS have open ports by default(at least common ones like : 80,443,etc.You can just change the port some common one so you don't have to do any port forwarding)

**Comment by ghost** [comment]
voOloX I was also trying to make a button on sripte to restart feeder but how to make any1 help plz

**Comment by GrinCa** [comment]
@VoOoLoX Thank's you very mych but I have just one question, Which file can I put on the server precisely ? And for info I don't code with Javascript, just C# et Java  ;).

**Comment by VoOoLoX** [comment]
@Pako98 Just clone the whole repo to the server and set it up like you would normally do, with the exceptions of adding the script and installing tampermonkey

**Comment by GrinCa** [comment]
ok I did everything I put the "Bots" folder on the server and have changed addresses. As against to start Start.bat I write "sh Start.sh" on putty but it is not recognized.

**Comment by GrinCa** [comment]
ok it's good I have solved the problem thank you for everything anyway

**Comment by GrinCa** [comment]
Excuse me for being boring but I do not understand why even on a server I only have 5 bots. Also I have a message in the console "Connection Failed: Error: Connection Timed Out" and I think that just ca ca. Do not think I'm a noob even if I am one;)

**Comment by VoOoLoX** [comment]
@Pako98 Get some better proxies
I update my proxies every day and i get ~150 bots, sometimes ~300 bots.

**Comment by GrinCa** [comment]
@VoOoLoX What do you mean by update proxies and how you doing? Is that what you think it impacts the number of bots because I do not see why.

**Comment by easyrider81** [comment]
The proxies get banned or just don't work over time so it's good practice to change to  fresh proxies in your proxy.txt file every few days.

@VoOoLoX regarding your comment:

> "--I'd suggest you to make a script to restart feeder server (because bots slowly disconnect after some time)"

If when the bots start to slowly disconnect can I start a second feeder.js to run at the same time as  the first to start connecting bots to the same server again or is this not possible?

**Comment by GrinCa** [comment]
@easyrider81 ok but How can you do that ( refresh the proxies ) ?  thank you ;)

**Comment by easyrider81** [comment]
@Pako98 Find new proxy addresses online and put them in the proxy.txt file. For example you could Google "free fast Socks5 proxy list" but you want to find good quality ones also. Quality is better than quantity.

**Comment by GrinCa** [comment]
ok thank you very much I'm again a noob in all this thing but I like learned on it ;)

**Comment by sejo153** [comment]
@Pako98 
Goods proxy here: :) http://proxylist.hidemyass.com/search-1302843#listable
And Proxy checker here: http://www.checker.freeproxy.ru/checker/

**Comment by GrinCa** [comment]
@sejo153 I look for good proxy but often when I run the .bat in the console me he wrote "Connection Failed: Error: read ECONNRESET" and no bot.
